config/config.form.yaml

```
@@ -47,6 +47,7 @@ snapshots:
 
 # docs in https://pypsa-eur.readthedocs.io/en/latest/configuration.html#enable
 enable:
```

Let's set it as last

```suggestion
enable:
  retrieve: auto
  retrieve_databundle: true
  retrieve_cost_data: true
  build_cutout: false
  retrieve_cutout: false
  custom_busmap: false
  drop_leap_day: true
  final_adjustment: true
```